src/utils.ts

```diff
@@ -58,6 +58,23 @@ export const ByteUtils = {
   }
 };
 
+/**
+ * Determines if a connection's address matches a user provided list
+ * of domain wildcards.
+ */
+export function hostMatchesWildcards(address: string, wildcards: string[]): boolean {
+  const host = HostAddress.fromString(address).host;
```

Comment:
This check should probably always pass for UNIX sockets? (If yes, that can be a follow-up as far as I am concerned.)

Reply:
I added the ability to check the unix socket now from the host address. Since it's not defined in the spec I've just assumed the simple "*/" leading wildcard for now and think it's fine for a follow up on the spec after.

Reply:
Alright, makes sense to me as a temporary solution to allow UNIX sockets :+1: